README.rst

```diff
@@ -21,7 +21,7 @@ GitHub_ page.
 Requirements
 ============
 
-Python >=3.6 (64-bit). (Note: Python 2 is not supported.)
+Python 3.7 (64-bit). (Note: Python 2 is not supported.)
```

Comment:
I think we can leave this. It definitely works on Python 3.8 (that's what Dan uses) and I've never see it fail on Python 3.6. The only reason I stopped it running on other versions is because we were approaching our free quota for CI tests.

Reply:
Will revert, but should revisit at a later date.

Reply:
Agree, would be nice

Reply:
Removed.